app/jobs/clock/refresh_subscription_usages_job.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module Clock
+  class RefreshSubscriptionUsagesJob < ApplicationJob
+    if ENV["SENTRY_DSN"].present? && ENV["SENTRY_ENABLE_CRONS"].present?
```

Comment:
worth it to maybe create `ClockJob` with these 7 lines it? 🤔

Reply:
@nudded, that's actually a great idea! Opening a PR for that